src/react-imgix.js

```diff
@@ -47,6 +48,26 @@ const SHARED_IMGIX_AND_SOURCE_PROP_TYPES = {
   src: PropTypes.string.isRequired
 };
 
+/**
+ * Parse an aspect ratio in the format w:h to a decimal. If false is returned, the aspect ratio is in the wrong format.
+ */
+function parseAspectRatio(aspectRatio) {
+  if (typeof aspectRatio == null) {
```

Comment:
This is potentially redundant.
It would be sufficient to make sure that `aspectRatio` is a string.
So check below would be sufficient on its own.